tools/benchmark/src/Reporter.ts

```diff
@@ -113,7 +113,7 @@ export class BenchmarkReporter {
     public constructor(outputDirectory?: string) {
         // If changing this or the result file logic in general,
         // be sure to update the glob used to look for output files in the perf pipeline.
-        this.outputDirectory = outputDirectory ?? path.join(__dirname, ".output");
+        this.outputDirectory = outputDirectory ? path.join('.', outputDirectory) : path.join(__dirname, ".output");
```

Comment:
@cqcmdwym - I was wondering why this prepends './' to the output directory?  I think this prevents users from using absolute paths (e.g., '/tmp/foo').

Reply:
@DLehenbauer Yes, I also have this concern when I thinking. I prefer to pass a relative path when using this option.
Do you have any other opinion? Thanks.

Reply:
I have use path.resolve function to handle both of the absolute and relative path.